### PR TITLE
Removal of code related to the Split Velocity Method

### PR DESCRIPTION
**Pull request by ArneVoss**
## Proposed Changes
To simulate a gust in CFD, typically the Field Velocity Method is used. This method prescribes a given velocity on each node and neglects any influence of the aircraft on the gust. The Split Velocity Method is an approach that allows to capture the influence of the aircraft on the gust. 

As far a I understand, the Split Velocity Method is only partially implemented in SU2 and isn't used for any gust type because all gust derivatives are zero / commented out in the code. This is in line with Padron et al. (https://doi.org/10.2514/6.2014-3013) who report that "... combining the GCL with the SVM does not work."

This pull request cleans up / removes the unused code.

## Related Work
The issue was identified during the work on pull request #1954

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [x] I am submitting my contribution to the develop branch.
- [x] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [ ] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [ ] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

File: SU2_CFD/src/iteration/CFluidIteration.cpp [hunk 1]
```diff
@@ -318,8 +313,8 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
   unsigned long iPoint;
   unsigned short iMGlevel, nMGlevel = config->GetnMGLevels();
 
-  su2double x, y, x_gust, dgust_dx, dgust_dy, dgust_dz, dgust_dt;
-  su2double *Gust, *GridVel, *NewGridVel, *GustDer;
+  su2double x, y, x_gust;
+  su2double *Gust, *GridVel, *NewGridVel;
```
Comment (pcarruscag) [inline SU2_CFD/src/iteration/CFluidIteration.cpp:317]: This is still not best practice but github doesn't let me comments on the lines I want to change xD
```suggestion
  su2double x, y, x_gust, Gust[3] = {0.0}, NewGridVel[3] = {0.0};
  const su2double* GridVel = nullptr;
```

File: SU2_CFD/src/iteration/CFluidIteration.cpp [hunk 2]
```diff
@@ -331,7 +326,6 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
 
   Gust       = new su2double[nDim]();
   NewGridVel = new su2double[nDim]();
```
Comment (pcarruscag) [inline SU2_CFD/src/iteration/CFluidIteration.cpp:328]:
```suggestion
```

File: SU2_CFD/src/iteration/CFluidIteration.cpp [hunk 3]
```diff
@@ -477,7 +441,6 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
   }
 
   delete[] Gust;
```
Comment (pcarruscag) [inline SU2_CFD/src/iteration/CFluidIteration.cpp:443]:
```suggestion
```

File: SU2_CFD/src/iteration/CFluidIteration.cpp [hunk 4]
```diff
@@ -477,7 +441,6 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
...
-  delete[] GustDer;
   delete[] NewGridVel;
```
Comment (pcarruscag) [inline SU2_CFD/src/iteration/CFluidIteration.cpp:444]:
```suggestion
```

**Review by pcarruscag — APPROVED** [review]

**Comment by ArneVoss** [comment]
Thank you for your corrections, Pedro, that's a quick pull request :)

**Pull request merged**